@xen-orchestra/web/src/pages/vm/new.vue

```diff
@@ -811,6 +812,42 @@ const vmData = computed(() => {
     ...(vdi.size && { size: giBToBytes(vdi.size) }),
   }))
 
+  function vifsToSend() {
+    const result = []
+    if (vmState.new_vm_template === undefined) return
```

Comment:
According to the guidelines, we have to use curly braces in this case
```suggestion
    if (vmState.new_vm_template === undefined) {
    
       return
    }
```